content/coronavirus_landing_page.yml

```diff
@@ -56,14 +56,17 @@ content:
     list:
       - heading: 25 December
         paragraph: |
-            You can make a Christmas bubble if you live in Tier 1, 2 or 3. You cannot make a Christmas bubble in Tier 4.
+            In Tier 4 areas in England, you cannot meet other people indoors unless you live with them or they are part of your support bubble. 
 
-            Follow guidance on [making a Christmas bubble with friends and family](/government/publications/making-a-christmas-bubble-with-friends-and-family/making-a-christmas-bubble-with-friends-and-family)
-      - heading: 20 December
+            You can make a Christmas bubble if you live in Tier 1, 2 or 3(/government/publications/making-a-christmas-bubble-with-friends-and-family/making-a-christmas-bubble-with-friends-and-family).
```

Comment:
The link here doesn't seem to be quite right - it seems to be missing the square brackets around the bits you are intending to link to.